ndmg/stats/qa_tractography.py

```diff
@@ -0,0 +1,113 @@
+    #Temporary list to store the open image:
+    im=[] 
+    #Temporary list to store the image nparray:
+    im_arr=[] 
+
+    fig= plt.figure()
+    plt.title(f'QA_tractography. Scan volume = {brain_ref_shape} \n\n', fontsize=60,fontweight='bold')
+    plt.xticks([])
+    plt.yticks([])
+    plt.axis("off")
+
+    j = 0
+    for i in range(9):
+        im.append(Image.open(qa_out_path + "/"+ plots_list[i]))
+        im_arr.append(np.array(im[i])) 
+        #Change the background of the image into black:
+        im_arr[i] = np.where(im_arr[i]<=0.01, 255, im_arr[i]) 
+        ax = fig.add_subplot(3,3,i+1)
+        ax.imshow(im_arr[i],interpolation="none",alpha=0.9)
+        if i<3:
+            ax.set_title(x_labels[i],fontsize=60,fontweight='bold')
+        if i % 3 == 0:
+            ax.set_ylabel(y_labels[j],fontsize=60,fontweight='bold')
+            j = j + 1
+        plt.xticks([])
+        plt.yticks([])
+            
+        im[i].close()
+    fig.set_size_inches(40, 40, forward = True)
+    fig.savefig(qa_out_path + "/" + 'qa_tractography.png', format='png')
+
+    for plot in plots_list:
+        os.chdir(qa_out_path)
+        #print(os.getcwd())
+        if os.path.exists(qa_out_path + "/" + plot):
+            os.remove(plot)
+        else:
+            print('No such file generated from streamlines window.')
+
+def qa_tractography(stream_path,qa_out_path,brain_path):
+    """
+    Visualize the streamlines, 
+    then using window.record to get nine snapshots of the 3D streamslines.
+
+    Parameters
+    ----------
+    stream_path : str
+        Path for the input streamline.trk file(s).
+    qa_out_path_path : str
+        Path for the output QA imgae file(s)
+    brain_path : str
+        Path for the reference brain scan, in order to get the scan volume.
+    """
+    
+    streamlines_mni = str(stream_path)
+    r = window.renderer()
+
+    streamlines_mni_load = nib.streamlines.load(streamlines_mni).streamlines
+    streamlines_mni_in = Streamlines(streamlines_mni_load)
+    streamlines_actor = actor.line(
+        streamlines_mni_in,
+        colormap.line_colors(streamlines_mni_in),
+        lod_points=10000,
+        depth_cue=True,
+        linewidth=0.2,
+        fake_tube=True,
+        opacity=0.3,
+    )
+    
+    r.add(streamlines_actor)
+    
+    interactive = False
+    if interactive:
+        window.show(r)
+        
+    showmng = window.ShowManager(r)
+    window.record(r,cam_pos = (70.03, 64.97, 269.80), cam_view = (0,1,0),path_numbering= True,out_path = qa_out_path + '/imgage1', az_ang=120,n_frames=3,reset_camera=True,size=(600,600))
+    window.record(r,cam_pos = (70.03, 64.97, 269.80), cam_view = (1,0,0),path_numbering= True,out_path = qa_out_path + '/imgage2', az_ang=120,n_frames=3,reset_camera=True,size=(600,600))
+    window.record(r,cam_pos = (70.03, 64.97, 269.80), cam_view = (0,0,1),path_numbering= True,out_path = qa_out_path + '/imgage3', az_ang=120,n_frames=3,reset_camera=True,size=(600,600))
```

Comment:
Give the images more descriptive names, image1,2,3 don't tell the user what each image is supposed to be.

Reply:
https://github.com/neurodata/ndmg/pull/375/commits/f9edd4a64206da4025ac3ece5970c649b197ff73